contracts/swaps/connectors/SwapsImplstETH_ETH.sol

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2017-2021, bZeroX, LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0.
+ */
+
+pragma solidity 0.5.17;
```

Comment:
also less restrictive if possible

Reply:
needs to be 0.5.17 because State.sol is 0.5.17